packages/storybook/src/Accordion/Accordion.stories.tsx

```diff
@@ -189,3 +182,209 @@ export function FullyControlled() {
     </Accordion>
   )
 }
+
+export function AccordionHookSingle() {
+  const {
+    setExpandedItems,
+    toggleExpanded,
+    checkIsExpanded,
+    register,
+    getHeaderProps,
+  } = useAccordion({
+    onChange: ({ expandedItems }) => console.info({ expandedItems }),
+    defaultExpandedItems: ['second'],
+    allowMultiple: false,
+  })
+
+  return (
+    <Flex sx={{ flexDirection: 'column', p: 6 }}>
+      <Accordion>
+        <AccordionItem>
+          <AccordionHeader
+            tabIndex={-1}
```

Comment:
This should be set in the hook.

Reply:
I was thinking that we don't know what the tabIndex should be if the user doesn't include something with a tabIndex inside, they might want the header to have the focus.

Reply:
I think we can always set `tabIndex={-1}` on the header and devs can add other elements inside that the user can tab to next.